frontend/src/App.tsx

```diff
@@ -1,14 +1,27 @@
-import React, { createContext } from "react";
+import React, { useState, createContext } from "react";
 import { Router, Switch, Route, Redirect } from "react-router-dom";
 import { createBrowserHistory } from "history";
 import HomePage from "./HomePage";
 import GameRouter from "./GameRouter";
+import Button from "./components/Button";
+import TextField from "./components/TextField";
+import PlayerList from "./components/PlayerList";
 import "./App.css";
 
 const browserHistory = createBrowserHistory();
 export const BrowserHistoryContext = createContext(browserHistory);
 
 function App() {
+  const [testText, setTestText] = useState("");
+
+  const clickBtn = () => {
+    console.log("Beep, boop");
```

Comment:
You may need to remove these console.log statements for the Github checks to pass

Reply:
For now I'm chucking `// eslint-disable-next-line` to mute the warning. This is obviously ugly but the components are just there for temporary testing and will be removed in my next PR.

Reply:
No worries!